deno.json

```diff
@@ -49,5 +47,46 @@
         "verbatim-module-syntax"
       ]
     }
-  }
+  },
+  "workspaces": [
```

Comment:
These have been sorted alphabetically to be easier to follow. Ditto for `browser-compat.tsconfig.json`.